tests/sync/session.cpp

```diff
@@ -138,10 +142,10 @@ TEST_CASE("SyncSession: management by SyncUser", "[sync]") {
     SECTION("a SyncUser properly rebinds existing sessions upon logging back in") {
         const std::string user_id = "user1d";
         auto user = SyncManager::shared().get_user(user_id, "not_a_real_token");
-        auto session1 = sync_session(server, user, "/test1b-1",
+        auto session1 = sync_session(server, user, "/test1d-1",
                                      [&](auto...) { return s_test_token; },
                                      [&](auto...) { });
-        auto session2 = sync_session(server, user, "/test1b-2",
+        auto session2 = sync_session(server, user, "/test1d-2",
                                      [&](auto...) { return s_test_token; },
```

Comment:
Yes. I did this for all occurrences I could find in 8c03a7d568b7fdefb5d2dddee141c2f49d0d9f91, but it looks like some new ones were added since. I think you may be able to get away with using `auto, auto`/`auto, auto, auto` for the argument types rather than being explicit about it like I was in that commit.